squad/frontend/templatetags/squad.py

```diff
@@ -95,6 +95,35 @@ def build_url(build):
     return reverse("build", args=(build.project.group.slug, build.project.slug, build.version))
 
 
+@register_global_function
+def previous_build_url(build):
+    previous_build = None
+    try:
+        previous_build = build.get_previous_by_created_at(project=build.project)
+    except Build.DoesNotExist:
+        pass
+    if previous_build:
+        return build_url(previous_build)
+
+
+@register_global_function
+def next_build_url(build):
+    next_build = None
+    try:
+        next_build = build.get_next_by_created_at(project=build.project)
+    except Build.DoesNotExist:
+        pass
+    if next_build:
+        return build_url(next_build)
+    else:
+        return build_url(build)
+
+
+@register_global_function
+def back_to_latest_build_url(build):
+    return build_url(Build.objects.filter(project=build.project).last())
```

Comment:
this might get a bit slow when there are a lot of builds. But I think it's OK to take this risk.